plugins/modules/aci_igmp_interface_policy.py

```diff
@@ -0,0 +1,367 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Tim Cragg (@timcragg) <tcragg@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_igmp_interface_policy
+short_description: Manage IGMP Interface Policies (igmp:IfPol)
+description:
+- Manage IGMP Interface Policies on Cisco ACI fabrics.
+options:
+  name:
+    description:
+    - The name of the IGMP Interface Policy
+    type: str
+  tenant:
+    description:
+    - Tenant to build the IGMP Interface Policy under. Defaults to common
+    type: str
+    default: common
+  description:
+    description:
+    - Description of the IGMP Interface Policy
+    type: str
+  group_timeout:
+    description:
+    - IGMP group timeout in seconds. APIC defaults to 260
```

Comment:
Could you consistently add an additional entry for the default in description for all arguments? See example below:

```
group_timeout:
    description:
    - IGMP group timeout in seconds.
    - The APIC defaults to 260 when unset during creation.